acceptance/tests/service_stop_start.rb

```diff
@@ -66,6 +78,20 @@ def assert_running
     assert_stopped
   end
 
+  if sysv?(@agent['platform'])
+    step '`service stop` stops all pxp-agent processes' do
+      start_service
+      assert_running
+      on(@agent, "rm -rf /var/run/puppetlabs/pxp-agent.pid")
+      # TODO remove after debugging
+      on(@agent, "cat /etc/rc.d/init.d/pxp-agent")
+      stop_service
```

Comment:
Ok, this test is failing because it uses Puppet to stop the service. But when Puppet queries `service pxp-agent status`, it's going to see that it's stopped and not do anything. So `service pxp-agent stop` is never called.

I think we could change this test to restart the service without puppet (since we know it's sysv, we can use `service pxp-agent restart`). The case we really care about is upgrades, and it's not Puppet restarting it then.